en/SUMMARY.md

```diff
@@ -1,152 +1,143 @@
+      * [struct EulerAngle](cpp/api_reference/structmavsdk_1_1_telemetry_1_1_euler_angle.md)
+      * [struct FixedwingMetrics](cpp/api_reference/structmavsdk_1_1_telemetry_1_1_fixedwing_metrics.md)
+      * [struct GpsInfo](cpp/api_reference/structmavsdk_1_1_telemetry_1_1_gps_info.md)
+      * [struct GroundTruth](cpp/api_reference/structmavsdk_1_1_telemetry_1_1_ground_truth.md)
+      * [struct Health](cpp/api_reference/structmavsdk_1_1_telemetry_1_1_health.md)
+      * [struct Imu](cpp/api_reference/structmavsdk_1_1_telemetry_1_1_imu.md)
+      * [struct MagneticFieldFrd](cpp/api_reference/structmavsdk_1_1_telemetry_1_1_magnetic_field_frd.md)
+      * [struct Odometry](cpp/api_reference/structmavsdk_1_1_telemetry_1_1_odometry.md)
+      * [struct Position](cpp/api_reference/structmavsdk_1_1_telemetry_1_1_position.md)
+      * [struct PositionBody](cpp/api_reference/structmavsdk_1_1_telemetry_1_1_position_body.md)
+      * [struct PositionNed](cpp/api_reference/structmavsdk_1_1_telemetry_1_1_position_ned.md)
+      * [struct VelocityNed](cpp/api_reference/structmavsdk_1_1_telemetry_1_1_velocity_ned.md)
+      * [struct VelocityBody](cpp/api_reference/structmavsdk_1_1_telemetry_1_1_velocity_body.md)
+      * [struct PositionVelocityNed](cpp/api_reference/structmavsdk_1_1_telemetry_1_1_position_velocity_ned.md)
+      * [struct Quaternion](cpp/api_reference/structmavsdk_1_1_telemetry_1_1_quaternion.md)
+      * [struct RcStatus](cpp/api_reference/structmavsdk_1_1_telemetry_1_1_rc_status.md)
+      * [struct StatusText](cpp/api_reference/structmavsdk_1_1_telemetry_1_1_status_text.md)
+    * [class Tune](cpp/api_reference/classmavsdk_1_1_tune.md)
+      * [struct TuneDescription](cpp/api_reference/structmavsdk_1_1_tune_1_1_tune_description.md)
+    * [namespace mavsdk (globals)](cpp/api_reference/namespacemavsdk.md)
+  * [Troubleshooting](cpp/troubleshooting.md)
   * [Contributing](contributing/README.md)
-    * [Building Source](contributing/build.md)
-    * [Testing](contributing/test.md)
+    * [Autogeneration](contributing/autogen.md)
     * [Writing Plugins](contributing/plugins.md)
-    * [Coding Style](contributing/code_style.md)
-    * [Plugin/Test Logging](contributing/dev_logging.md)
     * [Documentation](contributing/documentation.md)
     * [Release process](contributing/release.md)
-  * [Troubleshooting](cpp/troubleshooting.md)
-* [Autogeneration](autogen/README.md)
-* [Releases](releases/README.md)
-
-## Helpful Shortcuts
-
-* [PX4 Developer Guide](https://dev.px4.io/master/en/)
-* [PX4 User Guide](https://docs.px4.io/master/en/)
-* [QGroundControl User Guide](https://docs.qgroundcontrol.com/en/)
-* [QGroundControl Developer Guide](https://dev.qgroundcontrol.com/en/)
-* [MAVLink Guide](https://mavlink.io/en/)
-* [Dronecode Camera Manager](https://camera-manager.dronecode.org/en/)
+* [iOS/Swift](swift/README.md)
+* [Python](python/README.md)
```

Comment:
You need to add all files here. The reason the Python quickstart has a link warning is that it is not being built, because it is not included here.

Reply:
Actually @julianoes I've added it to easy review :-)

Reply:
Thx.